samples/net/gsm_modem/prj.conf

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
Better to put reel board specific stuff into `board/reel_board.conf` like we have in other samples.

Reply:
Let's remove FRDM board clutter, but leave Reel board stuff. Right now we haven't really enabled it elsewhere and without an existing UART one can't really run the sample anyway.

Reply:
Reel Board things are now in boards/